src/OData.QueryBuilder/Expressions/Visitors/ODataOptionFilterExpressionVisitor.cs

```diff
@@ -173,6 +173,10 @@ protected override string VisitMethodCallExpression(MethodCallExpression methodC
                     var indexOf1 = VisitExpression(methodCallExpression.Arguments[1]);
 
                     return $"{nameof(IODataFunction.IndexOf).ToLowerInvariant()}({indexOf0},{indexOf1})";
+                case nameof(IODataStringAndCollectionFunction.Length):
+                    var length0 = VisitExpression(methodCallExpression.Arguments[0]);
+
+                    return $"{nameof(IODataFunction.Length).ToLowerInvariant()}({length0})";
```

Comment:
`IODataStringAndCollectionFunction.Length` - So it would be more correct

Reply:
I agree, but all the other cases did the same, so I just went with it 😄